examples/pytorch/MNIST/example1/mnist_autolog_example1.py

```diff
@@ -217,11 +217,12 @@ def optimizer_step(
 
 
 if __name__ == "__main__":
+    mlflow.set_tracking_uri("http://localhost:5000/")
```

Comment:
It will be better to make the URL a config parameter so that user's don't need to change code if MLflow is running somewhere else